### PR TITLE
Fixes 08.08.2023

### PR DESCRIPTION
**Pull request by Oleg-Pecheneg**
Fixed channel's socket connection.
Ensure only interactable domains can be used for messaging. 
Updated channel's feed loading logic

### CODE REVIEW

File: unstoppable-ios-app/domains-manager-ios/Modules/DomainsCollection/DomainsCollectionViewController.swift [hunk 1]
```diff
@@ -686,24 +688,27 @@ private extension DomainsCollectionViewController {
...
         addBarButton.tintColor = .foregroundDefault
         addBarButton.accessibilityIdentifier = "Domains Collection Plus Button"
         
-        let messagingButton = DomainsCollectionMessagingBarButton()
-        messagingButton.pressedCallback = { [weak self] in self?.didTapMessagingButton() }
-        self.messagingButton = messagingButton
-        let messagingBarButton = UIBarButtonItem(customView: messagingButton)
-        
-        navigationItem.rightBarButtonItems = [addBarButton, messagingBarButton]
+        if canMessaging {
```
Comment (rommex) [inline unstoppable-ios-app/domains-manager-ios/Modules/DomainsCollection/DomainsCollectionViewController.swift:702]: Better 'canMessage' or something like  'isEnabledMessaging'

File: unstoppable-ios-app/domains-manager-ios/Modules/Messaging/ChatsList/Views/ChatsListNavigationView.swift [hunk 2]
```diff
@@ -237,15 +237,17 @@ struct ChatsListNavigationView_Previews: PreviewProvider {
             let wallet = WalletDisplayInfo(name: "name.x",
```
Comment (rommex) [inline unstoppable-ios-app/domains-manager-ios/Modules/Messaging/ChatsList/Views/ChatsListNavigationView.swift:237]: Is it mocked data? Such things should not be in prod code

File: unstoppable-ios-app/domains-manager-ios/Services/MessagingService/SubServices/ChannelsWebsockets/PushMessagingChannelsWebSocketsService.swift [hunk 3]
```diff
@@ -243,19 +243,20 @@ private extension PushMessagingWebSocketsService {
                 self?.onAny?(event)
             }
             
-            chatsConnection.socketManager.defaultSocket.onAny { [weak self] event in
-                self?.onAny?(event)
-            }
+            // Disabled for now
```
Comment (rommex) [inline unstoppable-ios-app/domains-manager-ios/Services/MessagingService/SubServices/ChannelsWebsockets/PushMessagingChannelsWebSocketsService.swift:246]: please MARK: when this needs to change

File: unstoppable-ios-app/domains-manager-ios/Modules/Messaging/ChatsList/Views/ChatsListNavigationView.swift (hunk 2 above)
Comment (Oleg-Pecheneg) [reply unstoppable-ios-app/domains-manager-ios/Modules/Messaging/ChatsList/Views/ChatsListNavigationView.swift:237]: This is SwiftUI preview, they're available only from Xcode

**Pull request merged**